docs/ops/movement/DepthToSpace_1.md

```diff
@@ -6,6 +6,26 @@
 
 **Short description**: *DepthToSpace* operation rearranges data from the depth dimension of the input tensor into spatial dimensions of the output tensor.
 
+**Detailed description**
+
+*DepthToSpace* operation permutes elements from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input depth dimension (features) `C` are moved to spatial blocks in `D1`, ..., `DK`. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#DepthToSpace) for an example of the 4D input tensor case.
+
```

Comment:
We generally avoid referring other frameworks, hence I'd remove this sentence with ONNX spec.

Reply:
done